### DISCUSSION TITLE
Does the 'log' config file option support flags? (e.g. '-v' for verbose)

### DISCUSSION THREAD

**Discussion by apitofme** [beetbox/beets#5182] [Q&A] [answered]
The [log](https://beets.readthedocs.io/en/stable/reference/config.html#log) config file option states only: _"Specifies a filename where the importer’s log should be kept..."_, however the default logging-level output is somewhat less informative than I would like.

```
import started Sat Apr  6 21:08:07 2024
import started Sat Apr  6 21:12:46 2024
import started Sat Apr  6 21:13:49 2024
import started Sat Apr  6 21:16:36 2024
...
```
As you can see from the sample log-file output above, it does not tell me anything about the processes going on behind the `import` command. I would like to see _verbose_ output in my log-file so I was curious if I could just add the `-v` flag to the `log: beets-log.txt` line in my `beets-config.yaml`, or do I have to specify it each time directly on the command-line?

Or is there some _other_ way to set the log-level verbosity that is persistent until changed?

**Comment by benpage2** [comment, marked as answer]
I don't think you can change the config to add verbose output to the `log:` config. It'll probably only accept a string for the file path.

The option `verbose:` (default `verbose: 0`) does exist in the default config, but it's not documented so changing it might help you not have to flag `-v` every time, or it might not. 

For reference, you can find the default config with all possible options (including some other undocumented ones) here https://github.com/beetbox/beets/blob/master/beets/config_default.yaml

**Reply by apitofme** [reply]
Yeah I tested passing the flag in the config and that doesn't work (I didn't really expect it to tbf), and the option isn't global as it's part of the `import` settings.

Thanks, sounds like the `verbose` option may be the way to go 👍
